tools/single_use/SaveUserIDToInstrumentData.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * This script is intended for a one-time use only to restore the value of the
```

Comment:
```suggestion
  * CAVEAT FOR PROJECTS: It can take up to a whole day to run this script. It took 14 hours to run the script on the CCNA DB during testing.
  *
  * This script is intended for a one-time use only to restore the value of the
```